### DISCUSSION TITLE
[Kavita] Problem with reverse proxy with a non-root URL path

### DISCUSSION THREAD

**Discussion by CubicPants** [Kareadita/Kavita#2864] [Ideas]
### Idea Description

Hi All,

It seems the site tries to pull down /assets/ instead of /kavita/assets/ when BaseURL='/kavita/;' I'm using Apache config as per the documentation. Any ideas?

Regards

### Idea Category

Web UI

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
Which repository are you using?

**Reply by scare376** [reply]
Oh this is a GitHub discussion. These are meant for feature requests primarily. I'm going to close it please create an issue or join us on discord to receive support there.

**Reply by CubicPants** [reply]
I used Docker Compose with this option:
`image: jvmilazz0/kavita:latest    # Using the stable branch from the offical dockerhub repo.
`
Docker is running as root user.

**Reply by CubicPants** [reply]
About System
Version
0.8.0.1
